docs/design/metrics/openkilda-metrics.md

```diff
@@ -0,0 +1,264 @@
+* `flowid` - String: ID of a sub flow which a rule belongs, `unknown` if there is no information about a subflow
+* `direction` - String: direction of flow traffic: `forward`, `reverse` or `undefined`
+* `is_y_flow_subflow` Boolean: `true` as a sub flow belongs to a Y-flow
+
+### Y-flow transit rule metrics
+Y-flow transit rules doesn't have some specific metrics, so kilda just stores [raw](#rule-metrics) metrics.
+
+### Y-flow egress rule metrics
+* `<prefix>.flow.bits` - total size of all packets which were processed by sub flow egress rule in bits
+* `<prefix>.flow.bytes` - total size of all packets which were processed by sub flow egress rule in bytes
+* `<prefix>.flow.packets` - total count of packets which were processed by sub flows egress rule
+
+Tags:
+* `y_flow_id` - String: ID of a Y-flow which a rule belongs, `unknown` if there is no information about a Y-flow
+* `flowid` - String: ID of a sub flow which a rule belongs, `unknown` if there is no information about a flow
+* `direction` - String: direction of flow traffic: `forward`, `reverse` or `undefined`
+* `is_y_flow_subflow` Boolean: `true` as a sub flow belongs to a Y-flow
+
+### Y-flow shared point rule metrics
+* `<prefix>.yFlow.shared.bits` - total size of all packets which were processed by Y-flow rule in shared point in bits
+* `<prefix>.yFlow.shared.bytes` - total size of all packets which were processed by Y-flow rule in shared point in bytes
+* `<prefix>.yFlow.shared.packets` - total count of packets which were processed by Y-flow rule in shared point
+
+Tags:
+* `y_flow_id` - String: ID of a Y-flow which a rule belongs, `unknown` if there is no information about a Y-flow
+* `flowid` - String: ID of a sub flow which a rule belongs, `unknown` if there is no information about a flow
+* `direction` - String: direction of flow traffic: `forward`, `reverse` or `undefined`
+* `is_y_flow_subflow` Boolean: `true` as a sub flow belongs to a Y-flow
+
+### Y-flow Y point rule metrics
+* `<prefix>.yFlow.yPoint.bits` - total size of all packets which were processed by Y-flow rule in Y point in bits
+* `<prefix>.yFlow.yPoint.bytes` - total size of all packets which were processed by Y-flow rule in Y point in bytes
+* `<prefix>.yFlow.yPoint.packets` - total count of packets which were processed by Y-flow rule in Y point
+
+Tags:
+* `y_flow_id` - String: ID of a Y-flow which a rule belongs, `unknown` if there is no information about a Y-flow
+* `flowid` - String: ID of a sub flow which a rule belongs, `unknown` if there is no information about a flow
+* `direction` - String: direction of flow traffic: `forward`, `reverse` or `undefined`
+* `is_y_flow_subflow` Boolean: `true` as a sub flow belongs to a Y-flow
+
+## HA-flow rule metrics
+### HA-flow ingress rule metrics
+* `<prefix>.haflow.ingress.bits` - total size of all packets which were processed by HA-flow ingress rule in bits
```

Comment:
here <prefix>.haflow.ingress.bits
should we use "haflow" instead of "haFlow" ?

just for comparison, for y-flow metrics, we use:
`<prefix>.yFlow.yPoint.bytes`

Reply:
lets discuss the name online